src/object/obj_rpc.c

```diff
@@ -1143,12 +1143,15 @@ int
 obj_reply_get_status(crt_rpc_t *rpc)
 {
 	void *reply = crt_reply_get(rpc);
-
+	
 	switch (opc_get(rpc->cr_opc)) {
 	case DAOS_OBJ_RPC_UPDATE:
 	case DAOS_OBJ_RPC_TGT_UPDATE:
 	case DAOS_OBJ_RPC_FETCH:
-		return ((struct obj_rw_out *)reply)->orw_ret;
+		{
+			D_DEBUG(DB_IO,"crt_reply: %d, DAOS_OBJ_RPC_UPDATE %d", (int) opc_get(rpc->cr_opc), (int)DAOS_OBJ_RPC_UPDATE);
```

Comment:
(style)  line over 100 characters
(style)  space required after that ',' (ctx:VxV)